index.js

```diff
@@ -29,24 +29,45 @@ function getPipeRiderOutputLog() {
   return '';
 }
 
+function getSummarySection(outputLog) {
+  var lines = outputLog.split('\n');
+  var summarySection = false;
+  var summary = [];
+  for(var i = 0;i < lines.length;i++){
+    if (lines[i].indexOf('Summary') > -1) {
```

Comment:
Suggest to match '── Summary ──', the table or column names could be Summary.

Reply:
fixed, thanks!